code/datums/ammo/ammo.dm

```diff
@@ -84,7 +84,7 @@
 	/// that will be given to a projectile with the current ammo datum
 	var/list/list/traits_to_give
 
-	var/flamer_reagent_type = /datum/reagent/napalm/ut
+	var/flamer_reagent_id = "utnapthal"
```

Comment:
Just a note: if we we required 515 this would be better as `var/flamer_reagent_id = /datum/reagent/napalm/ut::id` otherwise we could set it like `flamer_reagent_id = initial((/datum/reagent/napalm/ut):id)` in new or Initialize. But I wouldn't worry about this since I don't expect us to be changing the reagent ids and it should be pretty obvious if it gets renamed and things break.